demo/lit/src/my-element.js

```diff
@@ -0,0 +1,131 @@
+import { LitElement, css, html } from 'lit'
+import litLogo from './assets/lit.svg'
+import viteLogo from '/vite.svg'
+
+/**
+ * An example element.
+ *
+ * @slot - This element has a slot
+ * @csspart button - The button
+ */
+export class MyElement extends LitElement {
+  static get properties() {
+    return {
+      /**
+       * Copy for the read the docs hint.
+       */
+      docsHint: { type: String },
+
+      /**
+       * The number of times the button has been clicked.
+       */
+      count: { type: Number },
+    }
+  }
+
+  constructor() {
+    super()
+    this.docsHint = 'Click on the Vite and Lit logos to learn more'
+    this.count = 0
+  }
+
+  render() {
+    return html`
+      <div>
+        <a href="https://vitejs.dev" target="_blank">
+          <img src=${viteLogo} class="logo" alt="Vite logo" />
+        </a>
+        <a href="https://lit.dev" target="_blank">
+          <img src=${litLogo} class="logo lit" alt="Lit logo" />
+        </a>
+      </div>
+      <slot></slot>
+      <div class="card">
+        <button @click=${this._onClick} part="button">
+          count is ${this.count}
+        </button>
+      </div>
+      <p class="read-the-docs">${this.docsHint}</p>
+    `
+  }
+
+  _onClick() {
+    this.count++
+  }
+
+  static get styles() {
+    return css`
+      :host {
+        max-width: 1280px;
+        margin: 0 auto;
+        padding: 2rem;
+        text-align: center;
+      }
```

Comment:
@borkdude lit has this `css`magic that returns this:
![image](https://github.com/squint-cljs/vite-plugin-squint/assets/3731434/5ce156fa-dd67-4150-ba21-e9464ad03bf5)

Do you think it would be worth adding the possibility of having a `^css` to a string?